region/maridia/outer.json

```diff
@@ -4640,13 +4814,59 @@
                     "canInsaneWalljump",
                     "canCWJ"
                   ],
+                  "obstacles": [
+                    {
+                      "id": "A",
+                      "requires": [],
+                      "additionalObstacles": ["B"]
+                    }
+                  ],
                   "note": [
                     "Have to do a running stationary spinjump from a precise spot, to do a CWJ with slightly more speed off the wall one tile further out.",
                     "This makes it possible to just barely walljump off the grapple block."
                   ]
+                },
+                {
+                  "name": "Suitless Water Shinecharge",
+                  "notable": false,
+                  "requires": [
+                    "canWaterShineCharge",
+                    "canShinechargeMovement",
+                    {"or": [
+                      {"canShineCharge": {
+                        "usedTiles": 33,
+                        "shinesparkFrames": 61,
+                        "excessShinesparkFrames": 7,
+                        "openEnd": 2
+                      }},
+                      {"and": [
+                        "canMidairShinespark",
+                        {"canShineCharge": {
+                          "usedTiles": 33,
+                          "shinesparkFrames": 56,
+                          "excessShinesparkFrames": 7,
+                          "openEnd": 2
+                        }}
+                      ]}
+                    ]},
+                    {"adjacentRunway": {
+                      "fromNode": 1,
+                      "usedTiles": 1.5,
+                      "overrideRunwayRequirements": true,
+                      "useFrames": 80
+                    }}
+                  ],
+                  "obstacles": [
+                    {
+                      "id": "B",
+                      "requires": []
+                    }
+                  ],
+                  "note": "Start the dash as close to the door transition as possible in order to have a larger window for storing the shinecharge.",
```

Comment:
Its unclear what this means with a runway of 1.5 tiles.
And I can't do it without a water stutter.  I need 6 air tiles to shortcharge with.